packages/stencil/components/switch/src/components/osds-switch/styles/osds-switch.typography.scss

```diff
@@ -0,0 +1,14 @@
+@import '~@ovhcloud/ods-theming/typography/ods-theming-typography';
+@import '~@ovhcloud/ods-theming/typography/ods-component-typography';
+@import './osds-switch.mixins';
+
+// Main CSS mixin for typography
+@mixin osds-switch-theme-typography() {
+  :host {
+    $typography-properties: ods-get-typography-properties(body, '400');
```

Comment:
According to XD sketches, we should be using subheading '100' instead of body '400': 16px, SemiBold, line-height of 20px.
Still, currently it does the job since there's a font-weight: 600 on osds-radio so...